yjit/yjit.mk

```diff
@@ -5,12 +5,17 @@ CARGO_VERBOSE_0 = -q
 CARGO_VERBOSE_1 =
 CARGO_VERBOSE = $(CARGO_VERBOSE_$(V))
 
-# Select between different build profiles with macro substitution
-.PHONY: yjit-static-lib
-yjit-static-lib: yjit-static-lib-$(YJIT_SUPPORT)
+YJIT_SRC_FILES = $(wildcard \
+	$(top_srcdir)/yjit/Cargo.* \
+	$(top_srcdir)/yjit/src/*.rs \
+	$(top_srcdir)/yjit/src/*/*.rs \
+	$(top_srcdir)/yjit/src/*/*/*.rs \
+	$(top_srcdir)/yjit/src/*/*/*/*.rs \
```

Comment:
Does `wildcard` not support something like `yjit/src/**/*.rs`?

Reply:
Once I tried but it didn't work.